src/isar/models/mission.py

```diff
@@ -5,15 +5,13 @@
 
 from isar.config import config
 from isar.models.mission_metadata.mission_metadata import MissionMetadata
-from robot_interface.models.inspection.inspection import Inspection
-from robot_interface.models.mission import STEPS
+from robot_interface.models.step import Step
 
 
 @dataclass
 class Mission:
-    mission_steps: List[STEPS]
+    mission_steps: List[Step]
```

Comment:
I suspect this will break the local mission planner and how it reads missions from a file.

The dacite package which is used to read from json requires that the subclasses are specified, otherwise there is no way for it to infer which steps a mission could consist of.